src/main/java/org/sunbird/profile/service/ProfileServiceImpl.java

```diff
@@ -2409,6 +2425,21 @@ public SBApiResponse bulkUploadV2(MultipartFile mFile, String orgId, String chan
 		return response;
 	}
 
+	private boolean keyExistsInProfileDetails(Object value, String key) {
+		if (value instanceof Map) {
+			Map<String, Object> mapValue = (Map<String, Object>) value;
+			return mapValue.containsKey(key);
+		} else if (value instanceof List) {
+			List<?> listValue = (List<?>) value;
+			for (Object obj : listValue) {
+				if (obj instanceof Map && ((Map<?, ?>) obj).containsKey(key)) {
+					return true;
+				}
+			}
+		}
```

Comment:
@anilkumarkammalapalli  - do through exception here mentioning unidentified key value is been configured in the approval field... so that we are 100% sure on what we configure will come to know if any other type is been added in future...